src/modules/navigator/navigator.h

```diff
@@ -298,7 +303,7 @@ class Navigator : public control::SuperBlock
 	vehicle_land_detected_s				_land_detected{};	/**< vehicle land_detected */
 	vehicle_local_position_s			_local_pos{};		/**< local vehicle position */
 	vehicle_status_s				_vstatus{};		/**< vehicle status */
-
+	vehicle_status_s				_previous_vstatus{}; /**< vehicle status of the previous iteration*/
```

Comment:
Are you only using the nav state?

Reply:
Store only the previous uint8_t nav state.

Reply:
Makes sense. I changed it.